include/deal.II/base/array_view.h

```diff
@@ -511,6 +558,17 @@ inline ArrayView<ElementType, MemorySpaceType>::ArrayView(
 
 
 
+template <typename ElementType, typename MemorySpaceType>
+inline ArrayView<ElementType, MemorySpaceType>::ArrayView(
+  const std::initializer_list<std::remove_cv_t<value_type>> &initializer)
+  DEAL_II_CXX20_REQUIRES(std::is_const_v<ElementType>)
+  : // use delegating constructor
+  ArrayView((initializer.size() > 0 ? initializer.begin() : nullptr),
+            initializer.size())
```

Comment:
You are handling the `size==0` case elsewhere anyway.
```suggestion
  ArrayView(initializer.begin(), initializer.size())
```

Reply:
Good point. So changed.